dataeng/resources/snowflake-microbachelors-send-coaching-data-itk.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -ex
+
+# setup
+cd $WORKSPACE/analytics-tools/snowflake
+pip install -r requirements/microbachelors.txt
+
+# Run the script twice. Once for the student report, then again for the Course report
+python send_coaching_data_itk.py \
+    --key_path $KEY_PATH \
+    --passphrase_path $PASSPHRASE_PATH \
+    --user $USER \
+    --account $ACCOUNT \
+    --report-type student \
+    --send True
```

Comment:
Should we default both _send_ params to _False_ for now?

Reply:
Yeah, that makes sense until we are ready to go.